src/User/Infra/Security/SecurityUserProvider.php

```diff
@@ -40,6 +40,17 @@ public function loadUserByUsername($username): UserInterface
         return $this->fromUser($user);
     }
 
+    public function loadUserById(string $id): UserInterface
+    {
+        try {
```

Comment:
i think we should implement this where needed, thus `Jwt\SecurityUserProvider::loadUserByUsernameAndPayload`. You can inject the user repository there as well  :)